springer_fields_mapping.md

```diff
@@ -0,0 +1,219 @@
+| **publication_info**     | <a href="#generic_parsing">generic_parsing</a> : <a href="#46">46</a> | journal_title  |   |
+| **--**                   |                                                                       | journal_volume |   |
+| **--**                   |                                                                       | year           |   |
+| **--**                   |                                                                       | journal_issue  |   |
+| **--**                   |                                                                       | artid          |   |
+| **--**                   |                                                                       | page_start     |   |
+| **--**                   |                                                                       | page_end       |   |
+| **--**                   |                                                                       | material       |   |
+| **abstracts**            | <a href="#enhancer">enhancer</a>:<a href="#47">47</a>                 | value          |   |
+| **acquisition_source**   | <a href="#enhancer">enhancer</a>:<a href="#48">48</a>                 | source         |   |
+| **--**                   |                                                                       | method         |   |
+| **--**                   |                                                                       | date           |   |
+| **copyright**            | <a href="#enhancer">enhancer</a>:<a href="#49">49</a>                 | holder         |   |
+| **--**                   |                                                                       | year           |   |
+| **record_creation_date** | <a href="#enricher">enricher</a>:<a href="#69">69</a>                 |                |   |
+| **titles**               | <a href="#enhancer">enhancer</a>:<a href="#52">52</a>                 | title          |   |
+| **--**                   |                                                                       | source         |   |
+| **--**                   |                                                                       |                |   |
+| **$schema**              | <a href="#enricher">enricher</a>:<a href="#68">68</a>                 |                |   |
+
+
+
+# [enricher](#enricher)
+|                     |               |                                                     |
+|---------------------|---------------|-----------------------------------------------------|
+| Reference           | Field         | Enricher                                            |
+| <p id="68">[68]</p> | schema        | <a href="_get_schema">_get_schema</a>               |
+| <p id="69">[69]</p> | arxiv_eprints | <a href="_get_arxiv_eprints">_get_arxiv_eprints</a> |
```

Comment:
change `<p>` tag to `<a>`..e.g `<p id="68">[68]</p>` => `<a href="#68" id="68">[68]</a>`..this way you can also share by a click-able link